src/games/strategy/engine/framework/ProcessRunnerUtil.java

```diff
@@ -101,13 +102,11 @@ public static void exec(final List<String> commands) {
       // we need to read the input stream to prevent possible
       // deadlocks
       final Thread t = new Thread(() -> {
-        try {
-          while (s.read() >= 0) {
-            // just read
-          }
-        } catch (final IOException e) {
-          ClientLogger.logQuietly(e);
+        Scanner scanner = new Scanner(s);
```

Comment:
Why send the output to standard out?
Are you sure there will be no blocking issues readying bytes vs whole lines?